app/src/main/java/org/oppia/app/activity/ActivityComponent.kt

```diff
@@ -47,6 +48,7 @@ interface ActivityComponent {
   fun inject(continuePlayingFragmentTestActivity: ContinuePlayingFragmentTestActivity)
   fun inject(explorationActivity: ExplorationActivity)
   fun inject(homeActivity: HomeActivity)
+  fun inject(helpActivity: HelpActivity)
```

Comment:
Shift to one line up to follow the Alphabetical order.

Reply:
Done